architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -432,4 +432,39 @@ to run:
 `oc delete istag/ruby:latest`
 ====
 
+[[insecure-registries]]
 
+=== Importing images from insecure registries
```

Comment:
@soltysh Title case for the heading? So "Importing Images from Insecure Registries".